Makefile

```diff
@@ -131,6 +131,12 @@ test_python_version: ## Check Python version, throw error if compilation would f
 	@.venv/bin/python ./scripts/test_python_version.py
 
 
+.PHONY: _check_venv_active
+_check_venv_active:
+	# Checking whether virtual environment was activated
+	@python3 -c "import sys; assert sys.base_prefix!=sys.prefix"
```

Comment:
this probably already exists in .common.makefile

Reply:
yes, it seems it is there. But that one is not included. I figured there is a reason for that, so I added it directly to this Makefile